src/Sylius/Behat/Context/Api/Admin/NotificationContext.php

```diff
@@ -32,6 +32,6 @@ public function __construct(ApiClientInterface $client)
      */
     public function iShouldBeNotifiedThatSomethingHappened(): void
     {
-        Assert::true($this->client->isCreationSuccessful(), 'Currency could not be created');
+        Assert::true($this->client->isCreationSuccessful(), 'Resource could not be created');
     }
```

Comment:
```
  /**
     * @Then I should be notified that it has been successfully created
     */
    public function iShouldBeNotifiedThatSomethingHappened(): void
    {
        Assert::true($this->client->isCreationSuccessful(), 'Currency could not be created');
        Assert::true($this->client->isCreationSuccessful(), 'Resource could not be created');
    }
```

`Is successfully created` or `could not be created` ? 😄

Reply:
This is an exception message, so reversed information is expected here.

Reply:
I would work on the method name, as it's too generic 😄 but fix for the next PR imo 🖖